packages/teams-js/src/private/chat.ts

```diff
@@ -21,61 +21,53 @@ export namespace chat {
    * @hidden
    * Hide from docs
    * --------------
-   * Allows the user to start or continue a conversation with each subentity inside the tab
+   * Allows the user to open a chat with a single user using a deep link and
```

Comment:
I think it would make sense to remove the deep link mentions from the comments. I think from the app dev perspective this isn't deep link related anymore: they don't pass in URLs and no longer need to know about them.

Otherwise these comments look *fantastic* to me. Love the detail!